libraries/AP_Logger/LogStructure.h

```diff
@@ -451,6 +451,7 @@ struct PACKED log_TERRAIN {
     float current_height;
     uint16_t pending;
     uint16_t loaded;
+    float correction;
```

Comment:
I think we should rename this to "reference offset" to be consistent.

Reply:
done